src/components/Detailpage/Detailpage.tsx

```diff
@@ -0,0 +1,64 @@
+import { Skeleton, Stack, Typography } from '@mui/material';
+import { spacings } from '../DesignSystem/DesignSystem';
+import { ColumnTypes, ZaaktypeResolvedT } from '../../types/types';
+import DataGrid from '../DataGrid/DataGrid';
+import Tabs from '../Tabs/Tabs';
+import DetailpageActions from './Actions';
+import { getInitialData } from '../DataGrid/utils';
+import useDataGrid from '../../hooks/useDatagrid';
+import { algmeenContentStructure } from './utils';
+import Content from '../Content/Content';
+
+const columnNames: ColumnTypes[] = ['title', 'bulkEditor', 'default'];
+
+export type DetailpageProps = {
+  loading: boolean;
+  zaaktype?: ZaaktypeResolvedT;
+};
+
+const Detailpage = ({ loading, zaaktype }: DetailpageProps) => {
```

Comment:
I think that this component would be quite tricky to make generic :thinking: 

Maybe it should be moved out of the `components` folder and placed closer to the `ZaaktypenView` component. What do you think?

Reply:
Yes this was something i did wrong. It is domain specific so all the logic should be on the `zaaktypenview`